quickstart.md

```diff
@@ -174,13 +174,85 @@ root@cluster01-master-0:/home/ubuntu# crictl  -r  unix:///run/containerd/contain
 
 ## Creating a CoCo workload using a pre-existing encrypted image
 
-We will now proceed to download and run an encrypted container image using the CoCo building blocks.
+We will now proceed to download and run a sample encrypted container image using the CoCo building blocks.
 
-* *TBD: based on https://github.com/confidential-containers/operator/issues/77* *
+A demo container image is provided at [docker.io/katadocker/ccv0-ssh](https://hub.docker.com/r/katadocker/ccv0-ssh).
+It is encrypted with [Attestation Agent](https://github.com/confidential-containers/attestation-agent)'s [offline file system key broker](https://github.com/confidential-containers/attestation-agent/tree/64c12fbecfe90ba974d5fe4896bf997308df298d/src/kbc_modules/offline_fs_kbc) and [`aa-offline_fs_kbc-keys.json`](./aa-offline_fs_kbc-keys.json) as its key file.
 
-Change the yaml to point to the sample container image
-The keys for this image should be in the rootfs
+We have prepared a sample CoCo operator custom resource that is based on the standard `ccruntime.yamlccruntime.yaml`, but in addition has the the decryption keys and configuration required to decrypt this sample container image. To test out creating a workload from the sample encrypted container image, we can take the following steps:
 
+### Swap out the standard custom resource for our sample
+
```

Comment:
It might be worth adding a line as to why the custom resource needs to be removed. Something like
"Support for multiple custom resources in not available in the current release. Consequently, if a custom resource already exists, then you'll need to remove it first before deploying a new one. "

Reply:
Thanks for the suggestion. I've added it in.